src/restarts/restarts_manager.hpp

```diff
@@ -49,7 +50,10 @@ class RestartsManager : public IRestartsManager
                                                         initializer::PHAREDict const& dict)
     {
         auto rMan = std::make_unique<RestartsManager>(Writer::make_unique(hier, model, dict));
-        if (dict.contains("write_timestamps")) // else is only loading not saving restarts
+        auto restarts_are_written = core::any(
+            core::generate([&](auto const& v) { return dict.contains(v); },
+                           std::vector<std::string>{"write_timestamps", "elapsed_timestamps"}));
+        if (restarts_are_written) // else is only loading not saving restarts
```

Comment:
I don't understand the comment.

Reply:
it means it's only setup for loading from restart, not creating any new restart dumps

Reply:
could it mean it's neither saving nor loading (i.e. start from scratch without checkpointing)?

Reply:
generally yes, but we only go in this function if there's any restarts [key in the simulation dict](https://github.com/PHAREHUB/PHARE/blob/3b4644dcdb33e5dff0a7164dcd98d7e7a8753065/src/simulator/simulator.hpp#L266C68-L266C68)

essentially we don't configure the C++ side of restart writing if there is no timestamps, and loading is handled in the hierarchy